fms/distributed/tensorparallel.py

```diff
@@ -44,6 +44,18 @@ def apply_embedding_tp(par_mod: nn.Embedding, mod: nn.Embedding, world_size, ran
     # print(f"For rank {rank}, we have the following weights: Base weight {mod.weight} bias {mod.bias}; Par weight {par_mod.weight}, bias {par_mod.bias}")
 
 
+def apply_moe_tp(par_mod: nn.Module, mod: nn.Module, param_names, world_size, rank):
```

Comment:
in most cases it's `rank, world_size` not `world_size, rank` so I think we should make this consistent:

```
(latest) [34.223.4.174 06th 15:19:44 ~/repos/newfms]$ grep -irn 'world_size, rank' fms  | wc -l
4
(latest) [34.223.4.174 06th 15:19:47 ~/repos/newfms]$ grep -irn 'rank, world_size' fms  | wc -l
21
(latest) [34.223.4.174 06th 15:19:57 ~/repos/newfms]$ cd ../pytorch/
(latest) [34.223.4.174 06th 15:20:03 ~/repos/pytorch]$ grep -irn 'rank, world_size' torch | wc -l
70
(latest) [34.223.4.174 06th 15:20:27 ~/repos/pytorch]$ grep -irn 'world_size, rank' torch | wc -l
4
```